### DISCUSSION TITLE
https://greendolphindance.com/2023/10/07/%E5%8D%9A%E5%AE%A2%E6%90%AD%E5%BB%BA%E5%B0%8F%E8%AE%B0/

### DISCUSSION THREAD

**Discussion by giscus** [greendolphindance/greendolphindance.github.io#3] [General]
# https://greendolphindance.github.io/2023/10/07/%E5%8D%9A%E5%AE%A2%E6%90%AD%E5%BB%BA%E5%B0%8F%E8%AE%B0/

Back to the rough ground!

https://greendolphindance.github.io/2023/10/07/%E5%8D%9A%E5%AE%A2%E6%90%AD%E5%BB%BA%E5%B0%8F%E8%AE%B0/

**Comment by greendolphindance** [comment]
test

**Comment by SunnyCloudYang** [comment]
可以把giscus代码块作为article的最后一个子元素，这样评论区和文章就是一样宽的了🥳
以及Revolver老师想要交换友链嘛，可以把您加进去👀

**Reply by greendolphindance** [reply]
啊啊啊感谢！立刻改了
想要交换！！你想显示成什么名字，我的话Revolver就好

**Reply by SunnyCloudYang** [reply]
就Origakid吧（虽然博客完全origami无关🙊）

**Reply by greendolphindance** [reply]
好！等我起床添加！

**Comment by hanlin-ren** [comment]
好耶！

**Reply by greendolphindance** [reply]
嘿嘿

**Comment by redsherry** [comment]
很好，想装修就看您这篇

**Comment by leanty** [comment]
你好，在*代码复制按钮教程*里面指的post.ejs，这个post.ejs是layout\下的，还是layout\_page下的？我都试了，但是并没实现代码块的copy键效果

**Reply by greendolphindance** [reply]
是后者。不知道是什么问题诶，有把clipboard.min.js放在相应的位置吗？

**Reply by linyin022** [reply]
var codeBlocks = document.querySelectorAll('article .code pre');
应该是获取class=code的元素  这样子就能正常复制代码块

**Reply by leanty** [reply]
@linyin022 感谢，这样就好了
